doc/rst/usingchapel/prereqs.rst

```diff
@@ -4,16 +4,11 @@
 Chapel Prerequisites
 ====================
 
-If you have a standard UNIX environment, a C/C++ compiler, some basic
-scripting languages, a GNU-compatible make, and awk installed you should
-have no problems getting started with Chapel.
+Chapel is designed to be portable to a variety of systems and only
+requires tools that are common on UNIX systems.
```

Comment:
it annoys me that this version implies that all dependencies are tools (vs libraries)

Reply:
Would changing the wording to  "software that is common on UNIX systems."  reduce the annoyance?   Also, I want to ask if the current wording is good enough?

Reply:
I got here too late, but another option could've been "features..." or "capabilities that are common..."